scripts/mdpow-pow

```diff
@@ -193,49 +193,49 @@ if __name__ == "__main__":
     parser.add_argument('--plotfile', dest="plotfile", default='none',
                         help="plot dV/dlambda to FILE; use png or pdf suffix to "
                         "determine the file type. 'auto' generates a pdf file "
-                        "DIRECTORY/dVdl_MOLID_pcw.pdf and 'none' disables it "
-                        "The plot function is only available for mdpow estimator,"
+                        "DIRECTORY/dVdl_MOLID_pow.pdf and 'none' disables it. "
```

Comment:
In  `scripts/mdpow-solvationenergy` we use the form _DIRECTORY/dVdl_<molname>_<solvent>.pdf_ . To homogenize it would be better to use here _DIRECTORY/dVdl_<molname>_pow.pdf_ .

Reply:
added in 1a496aa7e66ca07b4630408b2d5bcb7dff0ad820